torchdynamo/symbolic_convert.py

```diff
@@ -293,6 +304,11 @@ def step(self):
             exc.real_stack.append(self.frame_summary())
             if self.empty_checkpoint():
                 raise
+        except Exception as exc:
```

Comment:
Should we log the stack here?

`logging` has a nice `logging.exception` function which also shows user the stack trace

Reply:
No we shouldn't. If we're inlining there may be many nested instruction translators, so we would print the exception multiple times. This is why I only print it at the top level. The other piece of complexity here is there are essentially two stack traces. The torchdynamo stack trace that is useful for us, and then the stack trace through the user code which should point to the line we were processing when the exception happened. (That's the `real_stack` shown here.